index.html

```diff
@@ -34,7 +34,7 @@
             <a class="nav-link" href="https://github.com/boutiques/boutiques/blob/master/examples/Getting%20Started%20with%20Boutiques.ipynb">Getting Started</a>
           </li>
           <li class="nav-item">
-            <a class="nav-link" href="https://brainhack101.github.io/neurolinks">Supported Tools</a>
+            <a class="nav-link" href="https://zenodo.org/">Supported Tools</a>
```

Comment:
This will just redirect to the Zenodo portal where all the tools are found. It would be better to use the URL used in `bosh search` to return only Boutiques tools instead.